test/plugin/Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY . .
 RUN mkdir -p /usr/local/lib/secretless && \
     go build -buildmode=plugin \
              -o /usr/local/lib/secretless/example-plugin.so \
-                 ./example/example_plugin.go
+                 ./example/
```

Comment:
I think you need the exact runner file specified or am I mistaken?

Reply:
when you modify it to use two files, you just reference the dir where the `main` package is defined. the automated tests should demonstrate that it works.